packages/flutter/lib/src/widgets/form.dart

```diff
@@ -571,13 +618,34 @@ class FormFieldState<T> extends State<FormField<T>> with RestorationMixin {
     super.deactivate();
   }
 
+  @override
+  void initState() {
+    super.initState();
+    if (widget.forceErrorText != null) {
+      // ignore: always_specify_types
```

Comment:
I think do `Future<void>` to avoid this `ignore`.

Reply:
Fixed it